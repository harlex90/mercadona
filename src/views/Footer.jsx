const Footer = () => {
    return(
        <div style={{backgroundColor: "red", width: "100%", height: "10%", position: "fixed", bottom: 0}}>
            <div style={{display: "flex", justifyContent: "center", alignItems: "end"}}>
                <h4>MERCADONA</h4>
            </div>
        </div>
    )
}


export default Footer;