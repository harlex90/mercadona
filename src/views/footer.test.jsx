import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
    test('match snapshot', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot()
    });
    test('display MERCADONA', () => {
        const { getByText } = render(<Footer />);
        expect(getByText("MERCADONA")).toBeInTheDocument();
    });
});



