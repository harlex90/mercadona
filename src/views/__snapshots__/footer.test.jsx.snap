// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer display MERCADONA 1`] = `
<DocumentFragment>
  <div
    style="display: flex; justify-content: center; align-items: end; background-color: red; width: 100%; height: 8%; margin-top: auto;"
  >
    <h4>
      MERCADONA
    </h4>
  </div>
</DocumentFragment>
`;

exports[`Footer match snapshot 1`] = `
<DocumentFragment>
  <div
    style="display: flex; justify-content: center; align-items: end; background-color: red; width: 100%; height: 8%; margin-top: auto;"
  >
    <h4>
      MERCADONA
    </h4>
  </div>
</DocumentFragment>
`;
